client/my-sites/plans/main.jsx

```diff
@@ -36,7 +36,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
> Let's force everyone to be explicit here and avoid any risk :)

Agreed. I'll remove the defaults and possibly throw an error somewhere if `displayJetpackPlans` is not defined in the component.